pyModeS/adsb.py

```diff
@@ -18,8 +18,7 @@
 """
 
 import math
-from . import util
-from .util import crc
+from pyModeS import util
```

Comment:
In my experience [its generally frowned upon to use relative imports](https://www.python.org/dev/peps/pep-0008/#imports), in fact, they are removed in Python 3. Its an example of the python guideline of being explicit instead of implicit. 

The issue you bring up about absolute imports incorrectly using an outdated or nonexistent package is avoided with the use of [virtual environments](https://virtualenv.pypa.io/en/stable/). With a virtualenv a python application will run with the version of python you specify and only install python packages specified in a config file (usually titled `requirements.txt`) into the virtualenv. Then when the application is run, it will only have access to its own code and the packages installed to the virtualenv, eliminating any chance of importing from the wrong package.

I'd like to ask that the change to absolute imports be kept, but if its going to break an integration you have, I can remove them and just keep the change to absolute imports in my fork. Please let me know 😃